### PR TITLE
Added loklak tweets

### PR DESCRIPTION
**Pull request by abishekvashok**
Added loklak tweets, work still in progress, the collabration is going on. Done my part, @PolBaldas it's your time to make the changes here's the url of the remote: `https://github.com/abhi2424shek/gci16.fossasia.org`

Further I added you as a collaborator at the same repo mentioned above.

### CODE REVIEW

**Comment by PolBaladas** [comment]
@Abhi2424shek gonna clone it now, do you know why id the Travis failing?

**Comment by abishekvashok** [comment]
I will add a commit to fix it you work now, I will fix it after the whole work is over, so we can also fix together the future errors!

**Comment by PolBaladas** [comment]
Okay!

**Comment by abishekvashok** [comment]
Sure 😉

**Comment by PolBaladas** [comment]
@Abhi2424shek Hi there, Are you reusing css classes from Last Issues section? Maybe you wanna rename them, change the class/css... If not the design will be dependant on the latest issues section.

**Comment by PolBaladas** [comment]
@Abhi2424shek did you end up doing the API part?

**Review by jigyasa-grover — CHANGES_REQUESTED** [review]
* Provide the link to live `gh-pages` from your forked repository or relevant screenshot for easier review.
* Refer the corresponding issue number in your commit message

**Comment by abishekvashok** [comment]
@jig08 we r on collaboration, its being made I will inform you when it is done.

**Comment by abishekvashok** [comment]
@gaeun welcome @PolBaladas mind incorporating her into as well @hemantjadon for this work you will accept us thrice right since, she has also claimed the task and we don't want races.

**Comment by hemantjadon** [comment]
@Abhi2424shek We want all of you to collaborate and make something awesome :) So no problems

File: javascripts/loklak-fetcher.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ *  loklak-fetcher-client
+ * 	by Yago González (C) 2016 - Under MIT license
+ * 	Bugs? Questions? Don't know what's the meaning of life?
+ * 	Take a look at: github.com/YagoGG/loklak-fetcher-client
+ *  Please, keep this header if you want to use this code. Thank you ;)
+ ******************************************************************************/
```
Comment (abishekvashok) [inline javascripts/loklak-fetcher.js:7]: See the header @PolBaladas

**Comment by abishekvashok** [comment]
We'll come back soon,,, :)

**Pull request closed without merging**